packages/medusa/src/commands/utils/open-admin.ts

```diff
@@ -0,0 +1,30 @@
+import { ConfigModule } from "@medusajs/types"
+import { getConfigFile } from "medusa-core-utils"
+import open from "open"
+
+/**
+ * A function to open the admin in the users default browser.
+ * @param port - The port the server is running on
+ * @param rootDirectory - The root directory of the project
+ * @returns Attempts to open the admin in the users default browser.
+ */
+export async function openAdmin(port: number, rootDirectory: string) {
+  // If we are not in development mode, we skip opening the admin
+  if (process.env.NODE_ENV !== "development") {
```

Comment:
When do we actually want to automatically open the admin? Doing so in production also doesn't make sense, as the backend will probably be deployed by some pipeline and therefore it will run headlessly.

Reply:
In dev only, but I have removed it for now, we need to find a better solution then the `open` package, as that one is not able to re-use tabs, and will open a new tab each time you restart your server.